scripts/explain_manifest/fixtures/ubuntu-22.04-amd64.txt

```diff
@@ -70,6 +70,11 @@
   - libc.so.6
   Runpath   : /usr/local/kong/lib
 
+- Path      : /usr/local/lib/lua/5.1/cjson.so
```

Comment:
Explanation: this feature depends on `lua-resty-ljsonschema`, which explicitly takes `lua-cjson` as a dependency, so we're now getting an additional `cjson.so` in the manifest. On my machine this file is a measly 42K, so we're not blowing up our package/image sizes.

If we want, I can put the `lua-resty-ljsonschema` addition in its own commit before merging so that it's more clear to future us why this is here.

Reply:
Update: I _did_ put this in its own commit, but feel free to squash if that makes more sense.